pangeo_forge_recipes/storage.py

```diff
@@ -39,8 +42,16 @@ def _copy_btw_filesystems(input_opener, output_opener, BLOCK_SIZE=10_000_000):
                     ) from e
                 if not data:
                     break
-                logger.debug(f"_copy_btw_filesystems copying block of {len(data)} bytes")
                 target.write(data)
+                bytes_read += len(data)
+                elapsed = time.time() - start
+                throughput = int(bytes_read / elapsed)
```

Comment:
```suggestion
                throughput = bytes_read / elapsed
```

Why does it have to be an int?

Reply:
It doesn't. When I was logging throughput as bytes/sec (as opposed to MB/sec), it seemed odd to include fractional bytes, and this was the simplest way I could think of to truncate the value. (In retrospect, string formatting might've been better.) Anyway, not needed either way anymore.